flow/Arena.cpp

```diff
@@ -102,17 +116,17 @@ bool Arena::hasFree(size_t size, const void* address) {
 }
 
 void ArenaBlock::addref() {
-	VALGRIND_MAKE_MEM_DEFINED(this, sizeof(ThreadSafeReferenceCounted<ArenaBlock>));
+	make_defined(this, sizeof(ThreadSafeReferenceCounted<ArenaBlock>));
```

Comment:
nit: With these now being functions rather than macros, we could use the typical camel case function naming scheme:

```suggestion
	makeDefined(this, sizeof(ThreadSafeReferenceCounted<ArenaBlock>));
```

Reply:
I also just noticed this was a draft, so apologies if I reviewed too soon.